server/ServerApp.h

```diff
@@ -292,6 +284,14 @@ class ServerApp : public IApp {
       * between two empires. Updates those empires of the change. */
     void    HandleDiplomaticMessageChange(int empire1_id, int empire2_id);
 
+    /**  Adds an existing empire to turn processing. The position the empire is
```

Comment:
why...?

Reply:
It is never used outside `ServerApp` and I decide to move it to private.

Reply:
OK